src/ApparentHorizons/ObjectLabel.hpp

```diff
@@ -8,7 +8,12 @@
 
 namespace ah {
 /// Labels for the objects in a binary system.
-enum class ObjectLabel { A, B };
+enum class ObjectLabel {
+  /// The object on the right side (along the grid-frame x-axis)
+  A,
+  /// The object on the left side (along the grid-frame x-axis)
```

Comment:
Maybe don't say right and left here because that's subjective to viewing angle. I know our convention is negative x on the left, but only saying positive/negative x-axis is unambiguous. How about just
```
The object along the positive/negative x-axis in the grid frame
```
?

Reply:
ok, done 👍